backend/core/types/src/backend-swagger.ts

```diff
@@ -3319,13 +3328,16 @@ export interface JurisdictionUpdate {
 
 export interface ListingFilterParams {
   /**  */
-  $comparison: EnumListingFilterParamsComparison
+  $comparison: EnumListingFilterParamsComparison;
 
   /**  */
-  name?: string
+  name?: string;
 
   /**  */
-  status?: EnumListingFilterParamsStatus
+  status?: ListingStatus;
+
+  /**  */
+  neighborhood?: string;
 }
```

Comment:
Something about my annotations on the listings controller GET endpoint are incorrect. I had to manually include these types because Swagger didn't.

Reply:
Updated based on @pbn4's comments below